lib/taskana-core-test/src/test/java/acceptance/jobs/HistoryCleanupJobAccTest.java

```diff
@@ -0,0 +1,523 @@
+package acceptance.jobs;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.sql.SQLException;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.function.ThrowingConsumer;
+import pro.taskana.TaskanaConfiguration.Builder;
+import pro.taskana.common.api.TaskanaEngine;
+import pro.taskana.common.internal.util.IdGenerator;
+import pro.taskana.common.internal.util.Pair;
+import pro.taskana.simplehistory.impl.SimpleHistoryServiceImpl;
+import pro.taskana.simplehistory.impl.jobs.HistoryCleanupJob;
+import pro.taskana.spi.history.api.TaskanaHistory;
+import pro.taskana.spi.history.api.events.task.TaskHistoryEvent;
+import pro.taskana.spi.history.api.events.task.TaskHistoryEventType;
+import pro.taskana.testapi.TaskanaConfigurationModifier;
+import pro.taskana.testapi.TaskanaInject;
+import pro.taskana.testapi.TaskanaIntegrationTest;
+import pro.taskana.testapi.WithServiceProvider;
+import pro.taskana.testapi.security.WithAccessId;
+
+@WithServiceProvider(
+    serviceProviderInterface = TaskanaHistory.class,
+    serviceProviders = SimpleHistoryServiceImpl.class)
+@TaskanaIntegrationTest
+class HistoryCleanupJobAccTest {
+
+  @TaskanaInject TaskanaEngine taskanaEngine;
+
+  SimpleHistoryServiceImpl historyService;
+
+  @WithAccessId(user = "businessadmin")
+  @BeforeAll
+  void setup() throws Exception {
+    historyService = new SimpleHistoryServiceImpl();
+    historyService.initialize(taskanaEngine);
+  }
+
+  @WithAccessId(user = "admin")
+  @BeforeEach
+  void before() throws Exception {
+    // clean previously created task histories
+    List<TaskHistoryEvent> taskHistoryEventList = historyService.createTaskHistoryQuery().list();
+    List<String> taskIds = taskHistoryEventList.stream().map(TaskHistoryEvent::getTaskId).toList();
+    if (!taskIds.isEmpty()) {
+      historyService.deleteHistoryEventsByTaskIds(taskIds);
+    }
+    assertThat(historyService.createTaskHistoryQuery().list()).isEmpty();
```

Comment:
Why do we need this assertion?

Reply:
You are right, we don't need it. I will delete it